easybuild/easyconfigs/z/Z3/Z3-4.12.5-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'CMakeMake'
```

Comment:
rebase without this ec

Reply:
fixed in 1c9177d8edfa006b4502d0bbd1311dcf8f60c361